Dockerfile

```diff
@@ -13,8 +13,8 @@ RUN ["./install-pdk-release.sh"]
 RUN apt-get remove -y curl && \
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/* && \
-    /opt/puppetlabs/pdk/private/ruby/2.4.5/bin/gem install --no-document onceover && \
-    ln -s /opt/puppetlabs/pdk/private/ruby/2.4.5/bin/onceover /usr/local/bin/onceover
+    /opt/puppetlabs/pdk/private/ruby/2.4.9/bin/gem install --no-document onceover && \
```

Comment:
Oooo, yeah that's annoying. I wonder if we can replace this with some sort of `$(find...)` subshell command.

Reply:
We could probably just install it into any Ruby version we find under `/opt/puppetlabs/pdk/private`, might want to test first to make sure it is installable under Ruby 2.1.9 though before spending too much time on that. :)

Reply:
Good call - got an error installing onceover using 2.1.9:

```
ERROR:  Error installing onceover:
        The last version of parallel (>= 0) to support your Ruby & RubyGems was 1.13.0. Try installing it with `gem install parallel -v 1.13.0` and then running the current command again
        parallel requires Ruby version >= 2.2. The current ruby version is 2.1.0.
```

So it looks like at least Ruby 2.2, at least until additional errors are uncovered (though I'm not sure why that says the current ruby version is 2.1.0...)

Reply:
> Oooo, yeah that's annoying. I wonder if we can replace this with some sort of `$(find...)` subshell command.

`$(dirname $(find /opt/puppetlabs/pdk/private/ruby/*/bin -type f -name 'pdk'))/gem install ...`